pkgs/build-support/autoconf-cache/default.nix

```
@@ -0,0 +1,18 @@
+{ stdenv, gettext, autoconf, automake }:
+
+stdenv.mkDerivation {
+  name = "autoconf-cache";
+  src = ./src;
+
+  buildInputs = [ gettext autoconf automake ];
```

```suggestion
  nativeBuildInputs = [ gettext autoconf automake ];
```

programs that are run during the build should be in nativeBuildInputs